test/core_tests.js

```diff
@@ -37,9 +37,10 @@
     });
 
     QUnit.test( "Attributes", function( assert ) {
-      assert.expect( 9 );
+      assert.expect( 10 );
 
       var actual, expected;
+      var root = document.querySelector( "div#impress" );
```

Comment:
Is there a reason for using `div#impress` instead of just `impress` in the selector? I suspect this tried to keep consistency but the line below seems to have selected the `html` by mistake, is that right?